software/controller/integration_tests/webcam/notes.txt

```diff
@@ -31,4 +44,19 @@ ffmpeg \
 Receive like this:
 ffplay udp://192.168.1.12:8081
 
-But this assumes direct streaming frrom 1.12 to 1.8 on LAN. How do we broadcast?
+Live streaming:
+- YouTube was chosen for live streaming the ventilator prototypes
+- Go to https://www.youtube.com/features?nv=1 and request "Live streaming"
+- Wait 24 hours for "Live streaming" to be enabled on your account
+- Go to YouTube Dashboard: https://www.youtube.com/live_dashboard?nv=1
+- Change category to "Nonprofits & Activism" and privacy to Unlisted
+- Copy "Stream name/key" and paste it in the KEY variable of the stream-to-youtube.sh script
+- Add execute permissions to the script: chmod +x stream-to-youtube.sh
+- Run the script with the KEY passed as the argument: ./stream-to-youtube.sh 8yuc-2eb2-74e9-tmb6-2cm6 (this is an example key)
+- Check that on the YouTube Dashboard that the status has gone from OFFLINE to STARTING to LIVE
+- The URL for the stream is available at the bottom of the page
+
+- There are two webcams are connected to Martin's setup, the server name is bigcheese.
```

Comment:
Redundant phrasing: "There are [...] are"

Reply:
Thanks for catching the typo. I corrected.